src/util.ts

```diff
@@ -2,46 +2,63 @@ import { AttachedScope, attachScopes } from '@rollup/pluginutils';
 import { Node, walk } from 'estree-walker';
 import isReference from 'is-reference';
 import type { AstNode } from 'rollup';
+import type { MemberExpression } from 'estree';
 
-const gmAPIs = [
-  'GM_info',
-  'GM_getValue',
-  'GM_getValues',
-  'GM_setValue',
-  'GM_setValues',
-  'GM_deleteValue',
-  'GM_deleteValues',
-  'GM_listValues',
-  'GM_addValueChangeListener',
-  'GM_removeValueChangeListener',
-  'GM_getResourceText',
-  'GM_getResourceURL',
-  'GM_addElement',
-  'GM_addStyle',
-  'GM_openInTab',
-  'GM_registerMenuCommand',
-  'GM_unregisterMenuCommand',
-  'GM_notification',
-  'GM_setClipboard',
-  'GM_xmlhttpRequest',
-  'GM_download',
-];
 const META_START = '// ==UserScript==';
 const META_END = '// ==/UserScript==';
+const GRANTS_REGEXP = /^unsafeWindow|GM[._][a-zA-Z0-9_]+/;
```

Comment:
Let's make it a little stricter? `/^(unsafeWindow$|GM[._]\w+)/`

Reply:
This also fixes an issue where `unsafeWindow` + other stuff would be matched. Nice 😄